blocks/form/mappings.js

```diff
@@ -19,7 +19,7 @@ export function getCustomComponents() {
  * Loads JS and CSS for a block.
  * @param {Element} block The block element
  */
-async function loadComponent(componentName, element, fd, container) {
+async function loadComponent(componentName, element, fd, container, formId) {
```

Comment:
Can't we just find the closest form element from the component  here and extract the form id?

Reply:
When field is getting init, its not yet attached to the form. So closest form is undefined at this moment in time @nit23uec

Reply:
Ah.. that's unfortunate. Passing the form id to all the methods that are required for html generation seems unintuitive.